lib/rucio/web/ui/templates/did.html

```diff
@@ -24,6 +24,13 @@ <h4>DID Metadata</h4>
     </table>
   </div>
 
+  <div id="metadata" class="columns panel">
+    <h4>Generic DID Metadata</h4>
+    <div id="loading">Loading, please wait...</div>
+    <table id="t_generic_metadata" style="word-wrap: break-word;">
```

Comment:
I guess we need to be careful with this. For now using the metadata functions will not work on the production instance (No table, no support for json) - But with this the webui will always call the did_meta method. I wonder if this will just crash everything. 
In general we need to carefully test this that running 1.18.0 (with this metadata support) with non-supportive databases will not crash everything.

Reply:
Also, on the webui side it will always show "Could not find DID". Which is a bit misleading. This div should be hidden by default and should only show when the generic did meta call is successful.